mne/__init__.py

```diff
@@ -1,3 +1,4 @@
+import bullshit
```

Comment:
I don't know this package ;)

Reply:
Dear @agramfort, yes, I think that sounds good. Sorry for not coming back earlier, I am currently busy but I think I can come back to this next week!

In the meantime, I took a programming workshop and the instructor helped me to get my local version of mne running in my own virtual environment. To test that it works we added an error by importing bullshit :D I'll remove that of course.